woothemes/woocommerce-rest-api-docs
# Updates based on issues in the WooCommerce repo

There are several issues in the backlog of the [REST API focus](https://github.com/woocommerce/woocommerce/labels/focus%3A%20rest%20api) in the WooCommerce repo. This attempts to batch fixes for those issues into one PR.

- [x] https://github.com/woocommerce/woocommerce/issues/43444
- [x] https://github.com/woocommerce/woocommerce/pull/46895 (add documentation for new refunds endpoint)
- [x] https://github.com/woocommerce/woocommerce/issues/28474
- [x] https://github.com/woocommerce/woocommerce/issues/45924
- [x] https://github.com/woocommerce/woocommerce/issues/45836
- [x] https://github.com/woocommerce/woocommerce/issues/43647
- [x] https://github.com/woocommerce/woocommerce/issues/35619
- [x] https://github.com/woocommerce/woocommerce/issues/32037

### To test

* After checking out this PR branch, run the script to build the static docs site files: `./build.sh`. This will create a `build` directory in the repo.
* Find the `index.html` file in the build directory and open it in a browser. You should see something that looks just like the [live REST API docs site](https://woocommerce.github.io/woocommerce-rest-api-docs/).
* Check for typos and other formatting issues related to the changes in this PR.
